packages/ecommerce-ui/components/FormComponents/CheckboxElement.tsx

```diff
@@ -55,7 +61,7 @@ export function CheckboxElement<TFieldValues extends FieldValues>({
                     {...rest}
                     color={rest.color || 'primary'}
                     sx={{
-                      ...rest.sx,
+                      ...sx,
```

Comment:
I don't think this'll work (not introduced in your change, just seeing it now):  https://mui.com/system/getting-started/the-sx-prop/#passing-the-sx-prop

```
        // You cannot spread `sx` directly because `SxProps` (typeof sx) can be an array.
        ...(Array.isArray(sx) ? sx : [sx]),
```